### PR TITLE
Create emails templating helper functions and spice up emails.

### PR DESCRIPTION
**Pull request by jtuckerk**
Trying to reference our style sheets from the api to keep the client and the email look and feel in sync was going to be too hard, so I've just manually created some templates within the api based of the client style.

I made some slight changes to the wording of some of the messages.

Styling is more limited on some mail clients than others, so in gmail for example you wont see the drop shadow

### CODE REVIEW

**Comment by jtuckerk** [comment]
image
image
image

**Comment by jtuckerk** [comment]
@trippersham for reference

File: api/notifications_api.py [hunk 1]
```diff
@@ -170,3 +181,58 @@ def minutes_difference(reset_req_time: datetime):
...
+                </table>
+            </td>
+        </tr>
+    </table>
+</body>
+"""
+
+# Try to keep this in sync with the style from the CSS element .btn and .btn-info in the client
+BUTTON_TEMPLATE = f"""<tr><td align="center">
+<a href="{{link}}" style="display: inline-block; font-weight: 400; color: #212529;
```
Comment (ecaoile) [inline api/notifications_api.py:226]: I'd personally add at least 1rem vertical margins on the button. Things looked slightly squished. Think something like `margin: 1rem auto` as one of the style props.

File: api/notifications_api.py [hunk 2]
```diff
@@ -170,3 +181,58 @@ def minutes_difference(reset_req_time: datetime):
     seconds_diff = abs((curr_time - reset_req_time).seconds)
     minutes_diff = seconds_diff/60
     return minutes_diff
+
+background_color = "#f3f2ef"
+card_background_color = "#ffffff"
+button_color = "#399DAC"
+# For email header image
+base_url = Config['routes']['client']
+if 'localhost' in base_url:
```
Comment (ecaoile) [inline api/notifications_api.py:190]: I'd personally do a ternary equivalent, but if you want the localhost reasoning to stick out, this is fine too.

**Review by ecaoile — APPROVED** [review]
Styling looks fabulous although I'd like some vertical spacing with the buttons.

File: api/notifications_api.py (hunk 2 above)
Comment (jtuckerk) [reply api/notifications_api.py:190]: I think I do want it to stick out more. I like how it reads this way. I should use the ternary operator more, but today isnt the day im gonna start

File: api/notifications_api.py (hunk 1 above)
Comment (jtuckerk) [reply api/notifications_api.py:226]: good call. done

**Pull request merged**